service/src/main/java/greencity/security/service/LoginAttemptServiceImpl.java

```diff
@@ -18,9 +22,8 @@ public class LoginAttemptServiceImpl implements LoginAttemptService {
 
     public LoginAttemptServiceImpl(HttpServletRequest request,
         @Value("${bruteForceSettings.blockTimeInHours}") long blockTimeInHours) {
-        super();
         this.request = request;
-        attemptsCache = CacheBuilder.newBuilder()
+        this.attemptsCache = CacheBuilder.newBuilder()
```

Comment:
we use for storing unsuccess events only in-memory cache
So it seems like after redeploying the app all data will be lost. Is it for the product owners?